core/app.py

```diff
@@ -80,6 +80,7 @@ def defaults(self):
     search_template = Template(filename='templates/settings/search.html', module_directory=core.MAKO_CACHE)
     quality_template = Template(filename='templates/settings/quality.html', module_directory=core.MAKO_CACHE)
     categories_template = Template(filename='templates/settings/categories.html', module_directory=core.MAKO_CACHE)
+    languages_template = Template(filename='templates/settings/languages.html', module_directory=core.MAKO_CACHE)
```

Comment:
Codacy found an issue: [Mako templates allow HTML/JS rendering by default and are inherently open to XSS attacks. Ensure variables in all templates are properly sanitized via the 'n', 'h' or 'x' flags (depending on context). For example, to HTML escape the variable 'data' do ${ data |h }.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6225754)